pygmt/io.py

```diff
@@ -0,0 +1,46 @@
+"""
+PyGMT input/output (I/O) utilities.
+"""
+import xarray as xr
+
+
+def load_dataarray(filename_or_obj, **kwargs):
+    """
+    Open, load into memory, and close a DataArray from a file or file-like
+    object containing a single data variable.
+
+    This is a thin wrapper around :py:meth:`xarray.open_dataarray`. It differs
+    from :py:meth:`xarray.open_dataarray` in that it loads the DataArray into
```

Comment:
Need to use `:py:func:` instead to link to https://xarray.pydata.org/en/stable/generated/xarray.open_dataarray.html#xarray.open_dataarray properly.
```suggestion
    This is a thin wrapper around :py:func:`xarray.open_dataarray`. It differs
    from :py:func:`xarray.open_dataarray` in that it loads the DataArray into
```